src/char/necro.py

```diff
@@ -0,0 +1,524 @@
+import keyboard
+from utils.custom_mouse import mouse
+from char import IChar
+from template_finder import TemplateFinder
+from ui import UiManager
+from pather import Pather
+from logger import Logger
+from screen import Screen
+from utils.misc import wait, rotate_vec, unit_vector
+import random
+from typing import Tuple, Union, List
+from pather import Location, Pather
+import numpy as np
+import time
+from utils.misc import cut_roi, is_in_roi
+
+import mouse as _mouse
+from mouse import _winmouse
```

Comment:
unused imports

Reply:
Yea I'll do another set of changes for static paths (teleport) once I get the gear for it setup. Mostly intended for walking, ill add a readme note on it  and in the parms. I'll take care of these changes then  should be about it!